language/diem-framework/modules/0L/Cases.move

```diff
@@ -29,10 +30,21 @@ address 0x1{
         public fun get_case(
             vm: &signer, node_addr: address, height_start: u64, height_end: u64
         ): u64 {
+            print(&3003122);
+            print(&height_start);
+            print(&height_end);
+            // this is a failure mode. Only usually seen in rescue missions, where epoch counters are reconfigured by writeset offline.
+            if (height_end < height_start) return VALIDATOR_DOUBLY_NOT_COMPLIANT;
+
             Roles::assert_diem_root(vm);
             // did the validator sign blocks above threshold?
+            print(&30031221);
+
             let signs = Stats::node_above_thresh(vm, node_addr, height_start, height_end);
+            print(&30031222);
```

Comment:
not a rustacean but i am wondering what these print statements do, they seem hardcoded. Is this a `print('got here1')` statement?